Added new test case for Linux VM hibernation with storage workload

juhlee-microsoft opened the pull request:
1. Added a new test case PERF-STORAGE-HIBERNATE
2. Fixed a few lines in POWER-HIBERNATE for VM size adjustment.

**TEST RESULT**
[LISAv2 Test Results Summary]
Test Run On           : 06/10/2020 07:56:21
ARM Image Under Test  : Canonical : UbuntuServer : 16.04-LTS : latest
Initial Kernel Version: 4.15.0-1083-azure
Final Kernel Version  : 5.7.0-next-20200609
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:48

   ID TestArea             TestCaseName                                                      TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------------------
    1 POWER                POWER-HIBERNATE                                                         PASS                45.99 
***
[LISAv2 Test Results Summary]
Test Run On           : 06/10/2020 07:56:30
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Initial Kernel Version: 5.3.0-1022-azure
Final Kernel Version  : 5.7.0-next-20200609
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:52

   ID TestArea             TestCaseName                                                      TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------------------
    1 POWER                POWER-HIBERNATE                                                         PASS                50.48 
***
[LISAv2 Test Results Summary]
Test Run On           : 06/10/2020 08:49:46
ARM Image Under Test  : Canonical : UbuntuServer : 16.04-LTS : latest
Initial Kernel Version: 4.15.0-1083-azure
Final Kernel Version  : 5.7.0-next-20200609
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:42

   ID TestArea             TestCaseName                                                      TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------------------
    1 POWER                POWER-STORAGE-WORKLOAD-HIBERNATE                                        PASS                40.65 
***
[LISAv2 Test Results Summary]
Test Run On           : 06/10/2020 08:49:50
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Initial Kernel Version: 5.3.0-1022-azure
Final Kernel Version  : 5.7.0-next-20200609
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:48

   ID TestArea             TestCaseName                                                      TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------------------
    1 POWER                POWER-STORAGE-WORKLOAD-HIBERNATE                                        PASS                46.03

juhlee-microsoft commented:
@ader1990 Is there anyother way than PPA? It is not secure and the packages in PPA is not validated by Canonical, https://help.ubuntu.com/community/PPA

ader1990 commented:
Execution tested:

```
 .\Run-lisav2.ps1 -TestPlatform Azure -RGIdentifier "tests-dpdk" -ARMImageName "Canonical UbuntuServer 18.04-LTS latest" -TestLocation "westus2" -CustomTestParameters "DPDK_SOURCE_URL=native" -TestNames "VERIFY-DPDK-COMPLIANCE"

Test Run On           : 03/07/2019 10:03:14
ARM Image Under Test  : Canonical : UbuntuServer : 18.04-LTS : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:6

   ID TestCaseName                                                                TestResult TestDuration(in minutes)
---------------------------------------------------------------------------------------------------------------------
    1 VERIFY-DPDK-COMPLIANCE                                                            PASS                 4.03
```

juhlee-microsoft committed:
Updated PerformanceTests.xml
Storage test needs bigger size VM for fio command
Update PERF-STORAGE-FIO-HB.ps1
New fio command with arguments
download log files
Need fio
Renamed fio output to json+ format
Replaced with timer constants

juhlee-microsoft committed:
Initial files check-in

juhlee-microsoft committed:
Added comments

juhlee-microsoft committed:
Merge branch 'master' into juhlee/hb_stor_perf

juhlee-microsoft committed:
Update ReplaceableTestParameters.xml

Change to master branch

juhlee-microsoft committed:
Update SetupHbKernel.sh

Missing dependency

juhlee-microsoft committed:
Update FunctionalTests-Power.xml

Renamed TC

juhlee-microsoft committed:
Update PerformanceTests.xml

Storage test needs bigger size VM for fio command

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

New fio command with arguments

juhlee-microsoft committed:
Update Power-Hibernate.ps1

download log files

juhlee-microsoft committed:
Update SetupHbKernel.sh

Need fio

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

Renamed fio output

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

Replaced with timer constants

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update SetupHbKernel.sh

juhlee-microsoft committed:
Merge branch 'juhlee/hb_stor_perf' of https://github.com/LIS/LISAv2 into juhlee/hb_stor_perf

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

Changed the path
VM size info update

juhlee-microsoft committed:
Update Power-Hibernate.ps1

VM size update

juhlee-microsoft committed:
New VM sizes in xml file

New VM size

juhlee-microsoft committed:
update

VM size change
fixed the fioCommand scripts

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

Print the current state

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

dcui reviewed (commented):
Looks good to me.

juhlee-microsoft committed:
Update Power-Hibernate.ps1

Disable Call trace detection for this test temporarily

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Moved hibernate command from kernel compile bash to PS script

juhlee-microsoft committed:
Update PERF-STORAGE-FIO-HB.ps1

juhlee-microsoft committed:
Merge branch 'master' into juhlee/hb_stor_perf

juhlee-microsoft committed:
Moved the test case to Functional

juhlee-microsoft committed:
Merge branch 'juhlee/hb_stor_perf' of https://github.com/LIS/LISAv2 into juhlee/hb_stor_perf

juhlee-microsoft committed:
Update FunctionalTests-Power.xml

LiliDeng reviewed (dismissed):
https://github.com/LIS/LISAv2/blob/master/Testscripts/Windows/Power-Hibernate.ps1#L127 need change.
And squash all commits into one when do merge.

juhlee-microsoft committed:
Update Power-Hibernate.ps1
